src/ansible_navigator/_yaml.py

```diff
@@ -45,10 +53,19 @@ def human_dump(obj: Any, filename: str = None, fmode: str = "w") -> Union[str, N
 
 class HumanDumper(Dumper):
     # pylint: disable=too-many-ancestors
-    """for block scalar for multiline"""
+    """An instance of a pyyaml Dumper.
+
+    This deviates from the base to dump a multiline string in a human readable format.
+    """
+
+    def represent_scalar(self, tag: str, value: str, style=Union[str, None]) -> yaml.ScalarNode:
+        """Use a block scalar for a nicer human representation of multiline strings.
```

Comment:
What does this do exactly? "use" does not specify a side-effect, is it a converter? Maybe it'd be "Convert one thing into another"?

Reply:
updated